pkg/metaconfig/metaconfig.go

```diff
@@ -0,0 +1,20 @@
+package metaconfig
+
+import "gopkg.in/yaml.v2"
+
+type MetadataConfig struct {
```

Comment:
I would argue for just Metadata. MetadataConfig seems to imply this is configuration of Metadata while it is actually just Metadata.

Reply:
agreed. i was struggling with this name.